.cargo/config

```diff
@@ -1,7 +1,28 @@
-[target.thumbv7em-none-eabihf]
-runner = 'arm-none-eabi-gdb'
+[target.thumbv7m-none-eabi]
+# uncomment this to make `cargo run` execute programs on QEMU
+# runner = "qemu-system-arm -cpu cortex-m3 -machine lm3s6965evb -nographic -semihosting-config enable=on,target=native -kernel"
+
```

Comment:
Can we get rid of these?

Reply:
Sure, sounds ok to me. Haven’t tested it either, the QEMU settings all came from one of the other BSCs where I got this from. Will clean it up later today.